<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="1" failures="2" hostname="PauloCÃ©sar" name="submitleda.RoteiroControllerSpec" skipped="0" tests="7" time="1.309" timestamp="2016-03-29T04:14:38">
  <properties />
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test the index action returns the correct model" time="0.534" />
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test the create action returns the correct model" time="0.13" />
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test the save action correctly persists an instance" time="0.141">
    <failure message="Condition not satisfied:&#xa;&#xa;response.redirectedUrl == &apos;/roteiro/show/1&apos;&#xa;|        |             |&#xa;|        null          false&#xa;org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@b32e983&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Condition not satisfied:

response.redirectedUrl == &apos;/roteiro/show/1&apos;
|        |             |
|        null          false
org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@b32e983

	at submitleda.RoteiroControllerSpec.Test the save action correctly persists an instance(RoteiroControllerSpec.groovy:57)
</failure>
  </testcase>
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test that the show action returns the correct model" time="0.077" />
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test that the edit action returns the correct model" time="0.082" />
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test the update action performs an update on a valid domain instance" time="0.197">
    <error message="Cannot get property &apos;id&apos; on null object" type="java.lang.NullPointerException">java.lang.NullPointerException: Cannot get property &apos;id&apos; on null object
	at submitleda.RoteiroControllerSpec.Test the update action performs an update on a valid domain instance(RoteiroControllerSpec.groovy:122)
</error>
  </testcase>
  <testcase classname="submitleda.RoteiroControllerSpec" name="Test that the delete action deletes an instance if it exists" time="0.111">
    <failure message="Condition not satisfied:&#xa;&#xa;Roteiro.count() == 1&#xa;        |       |&#xa;        0       false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Condition not satisfied:

Roteiro.count() == 1
        |       |
        0       false

	at submitleda.RoteiroControllerSpec.Test that the delete action deletes an instance if it exists(RoteiroControllerSpec.groovy:142)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from Test the index action returns the correct model--
--Output from Test the create action returns the correct model--
--Output from Test the save action correctly persists an instance--
--Output from Test that the show action returns the correct model--
--Output from Test that the edit action returns the correct model--
--Output from Test the update action performs an update on a valid domain instance--
--Output from Test that the delete action deletes an instance if it exists--
]]></system-out>
  <system-err><![CDATA[--Output from Test the index action returns the correct model--
--Output from Test the create action returns the correct model--
--Output from Test the save action correctly persists an instance--
--Output from Test that the show action returns the correct model--
--Output from Test that the edit action returns the correct model--
--Output from Test the update action performs an update on a valid domain instance--
--Output from Test that the delete action deletes an instance if it exists--
]]></system-err>
</testsuite>
